components/List/ListPoster.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import { Image } from 'expo-image';
+import FontAwesome6 from '@expo/vector-icons/FontAwesome6';
+import { getPosterUrl } from '../../constants';
+import { LIST_PLACEHOLDER } from '../../constants';
+
+interface PosterProps {
```

Comment:
We using `interface` in some places and `type` in others, should we be consistent and pick one?